python/python/glide/async_commands/core.py

```diff
@@ -1202,6 +1224,42 @@ async def zcard(self, key: str) -> int:
         """
         return cast(int, await self._execute_command(RequestType.Zcard, [key]))
 
+    async def zcount(
+        self,
+        key: str,
+        min_score: Union[InfBound, BoundPair],
+        max_score: Union[InfBound, BoundPair],
+    ) -> int:
+        """
+        Returns the number of members in the sorted set stored at `key` with scores between `min_score` and `max_score`.
+
+        See https://redis.io/commands/zcount/ for more details.
+
+        Args:
+            key (str): The key of the sorted set.
+            min_score (Union[InfBound, BoundPair]): The minimum score to count from.
+                Can be an instance of InfBound representing positive/negative infinity,
+                or BoundPair representing a specific score and inclusivity.
+            max_score (Union[InfBound, BoundPair]): The maximum score to count up to.
+                Can be an instance of InfBound representing positive/negative infinity,
+                or BoundPair representing a specific score and inclusivity.
+
+        Returns:
+            int: The number of members in the specified score range.
+
+        Examples:
+            >>> await zcount("my_sorted_set", BoundPair(5.0 , true) , InfBound.POS_INF)
```

Comment:
please add the arg names to the BoundPair example to make it more readable, e.g.:
```
BoundPair(value=5.0 , is_inclusive=true)